http/getAxiosConfig.ts

```diff
@@ -4,6 +4,26 @@ import { getHubSpotApiOrigin } from '../lib/urls';
 import { AxiosConfigOptions } from '../types/Http';
 import { CLIConfig } from '../types/Config';
 import { AxiosRequestConfig } from 'axios';
+import https from 'https';
+import http from 'https';
+
+// Total number of sockets across all hosts
+const MAX_TOTAL_SOCKETS = 25;
```

Comment:
We could probably be more conservative with this number since this means there could be 25 idle socket connections in the worst case scenario.  The vast majority of our API calls seem to be to hubspot domains, so the likely hood of that happening feels slim.  Still, curious what others think this.